stubs/docutils/docutils/parsers/rst/states.pyi

```diff
@@ -1,6 +1,97 @@
 from _typeshed import Incomplete
+from collections.abc import Callable, Sequence
+from re import Match, Pattern
+from types import ModuleType
+from typing import Any
+from typing_extensions import TypeAlias
+
+from docutils import nodes
+from docutils.utils import Reporter
+
+class Struct:
+    def __init__(self, **keywordargs) -> None: ...
```

Comment:
```suggestion
    def __init__(self, **keywordargs) -> None: ...
    def __getattr__(self, attr: str, /) -> Any: ...
```
Since it puts all the kwargs in its `__dict__`.